examples/slu/slurp/scripts/download_data.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
```

Comment:
No recipes like this please. Convert it to a python script that can be run form the scripts/dataset dir in nemo root.